### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#10] [Ideas]
Display and/or export all the voices in a table format.

**Comment by BobanSpasic** [comment]
Hi.
I can do it, it is no problem, but what is the practical purpose?

**Reply by gigli69** [reply]
It is to visualize quickly all the parameters of the different voices, to be able to make comparison, sort, ...
And to better understand how to build the sounds. 
One pratical use is that I want to try to recreate the PSS sounds on a TG77. :)

**Reply by BobanSpasic** [reply]
Martin Tarenskeen tried to make a converter from PSS to DX7, but...
If the target platform does not have the basic wave shapes needed (sine, half-sine, squared-sine, half-squared-sine), you have no luck.
DX7 has just the sine wave. Even DX11 has more (7 or 8 basic wave shapes, can't recall right now). I know that TG77 has even more, but are all 4 wave shapes from PSS included?

I also wanted to re-create the Tremolo Guitar on a DX7, but no luck. It was my favorite voice on the PSS-780.

Anyway, I'll do something about what you asked. Exporting to CSV files does the job for you?
Simply make a sketch of what you need, and I'll try to implement it for you.
